kubernetes/omsagent.yaml

```diff
@@ -379,13 +379,19 @@ spec:
               memory: 225Mi
           env:
             # azure devops pipeline uses AKS_RESOURCE_ID and AKS_REGION hence ensure to uncomment these
+            - name: AKS_CLUSTER_NAME
+              value: "VALUE_AKS_CLUSTER_NAME"
             - name: AKS_RESOURCE_ID
               value: "VALUE_AKS_RESOURCE_ID_VALUE"
             - name: AKS_REGION
               value: "VALUE_AKS_RESOURCE_REGION_VALUE"
+            - name: FBIT_SERVICE_FLUSH_INTERVAL
+              value: "15"
+            - name: AKS_NODE_RESOURCE_GROUP
+              value: "VALUE_AKS_NODE_RESOURCE_GROUP"
             # this used for e2e test and setting this just emits some additional log statements which used for the e2e tests
-            - name: ISTEST  
-              value: "true"
+            #- name: ISTEST  
```

Comment:
what's the reason for commenting this ? This used for e2e test - https://github.com/microsoft/Docker-Provider#e2e-tests. If you comments these, e2e test will fail.

Reply:
Uncommenting this,